public/css/tillScreen.css

```diff
@@ -0,0 +1,177 @@
+body {
+  overflow-x: hidden;
+  overflow-y: hidden;
+}
+
+select {
+  width: 10em;
+}
+
+.x {
+  padding-left:.2em;
+  padding-right:.2em;
+  padding-top:.1em;
+  padding-bottom:.1em;
+  font-size: .7em;
+}
+
+#tillNumbers {
+  position:absolute;
```

Comment:
You want to avoid absolute positioning wherever possible - there's pretty much always a better way of making your CSS work. 

More reading: http://demosthenes.info/blog/135/CSS-Positioning-absolute-the-overused